test/shared_utils/compare.py

```diff
@@ -38,6 +55,10 @@ def compare_results_to_reference_file(
     reference_data = DataTree.from_dict(reference_groups)
     results_data = DataTree.from_dict(results_groups)
 
+    # Limit comparison of data
+    if subset_selector is not None:
+        results_data = subset_datatree(results_data, subset_selector)
```

Comment:
Okay, again just asking basic things for clarification:

In the instance that `subset_selector` is specified, you're saying the output results file needs to be subsetted, _but_ that it is assumed that the reference file already has that subsetting applied to it.

If that's the case, it might be worth mentioning that in the documentation string. (i.e. that the reference file is expected to already have such subsetting applied)

Reply:
Yes, I could leave the entire datatree in the reference file and subset both, but they are just a little bit smaller if they're stored subsetted.

Reply:
1a90cc2